packages/next-crawler-utils/tsconfig.json

```
@@ -0,0 +1,10 @@
+{
+  "compilerOptions": {
+    "module": "commonjs",
+    "target": "es2015",
```

```suggestion
    "target": "es2019",
```

I think this can be Node.js 12+ since its server-side only